tests/piqueserver/test_statistics.py

```diff
@@ -0,0 +1,37 @@
+"""
+test piqueserver/server.py
+"""
+
+from twisted.trial import unittest
+from unittest.mock import Mock
```

Comment:
`unittest.mock` was introduced in python3.3 - need to use something different or drop python2 support

Reply:
that's annoying... I thought I tested it with trials